trapper/commands.py

```diff
@@ -94,6 +103,90 @@ def add_subparser(
         return subparser
 
 
+@Subcommand.register("distributed-run")
+class DistributedRun(Subcommand):
+    """trapper's run command that enables running an experiment in
+    a distributed way.
+
+    Usage:
+        Basic:
+            ` trapper distributed-run experiment.jsonnet <--other-kwargs vals>`
+    """
+
+    def add_subparser(
+        self, parser: argparse._SubParsersAction
+    ) -> argparse.ArgumentParser:
+        description = """Distributed run the experiment specified in the config file which
```

Comment:
I suggest phrasing this sentence better e.g. something like "Start an experiment in a distributed parallel fashion using the config file ..."